docs/tutorials/sensormanagement/03_OptimisedSensorManagement.py

```diff
@@ -0,0 +1,629 @@
+    tracksC.append(Track([prior]))
+
+# %%
+# Create sensor managers
+# ^^^^^^^^^^^^^^^^^^^^^^
+#
+# The :class:`~.UncertaintyRewardFunction` will be used for each sensor manager as in Tutorials 1 & 2
+# and the :class:`~.BruteForceSensorManager` will be used as a comparison to the optimised methods.
+
+from stonesoup.sensormanager.reward import UncertaintyRewardFunction
+from stonesoup.sensormanager import BruteForceSensorManager
+
+# %%
+# Optimised Brute Force Sensor Manager
+# """"""""""""""""""""""""""""""""""""
+#
+# The first optimised method, :class:`~.OptimizeBruteSensorManager` uses :func:`~.scipy.optimize.brute`
+# which minimizes a function over a given range using a brute force method. This can be tailored by setting
+# the number of grid points to search over or by adding the use of a polishing function.
+
+from stonesoup.sensormanager import OptimizeBruteSensorManager
+
+# %%
+# Optimised Basin Hopping Sensor Manager
+# """"""""""""""""""""""""""""""""""""""
+#
+# The second optimised method, :class:`~.OptimizeBasinHoppingSensorManager` uses :func:`~.scipy.optimize.basinhopping`
+# which finds the global minimum of a function using the basin-hopping algorithm. This is a combination of a
+# global stepping algorithm and local minimization at each step. Parameters such as number of basin hopping
+# iterations or stepsize can be set to tailor the algorithm to requirements.
+
+from stonesoup.sensormanager import OptimizeBasinHoppingSensorManager
+
+# %%
+# Initiate sensor managers
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Create an instance of each sensor manager class. For the optimised sensor managers the default settings
+# will be used meaning only a sensor set and reward function is required. The :class:`~.UncertaintyRewardFunction`
+# will be used for each sensor manager.
+
+
+# initiate reward function
+reward_function = UncertaintyRewardFunction(predictor, updater)
+
+bruteforcesensormanager = BruteForceSensorManager(sensor_setA,
+                                                  reward_function=reward_function)
+
+optimizebrutesensormanager = OptimizeBruteSensorManager(sensor_setB,
+                                                        reward_function=reward_function)
+
+optimizebasinhoppingsensormanager = OptimizeBasinHoppingSensorManager(sensor_setC,
+                                                                      reward_function=reward_function)
+
+# %%
+# Run the sensor managers
+# ^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Each sensor management method requires a timestamp and a list of tracks at each time step when calling
+# the function :meth:`choose_actions`. This returns a mapping of sensors and actions to be taken by each
+# sensor, decided by the sensor managers.
+#
+# For each sensor management method, at each time step the chosen action is given to the sensors and then
+# measurements taken. The tracks are updated based on these measurements with predictions made for tracks
+# which have not been observed.
+#
```

Comment:
The paragraph above loosely implies that predictions haven't been made for the tracks that are observed. A minor re-wording could help clarify this.